### ISSUE TITLE
Vets-api terminal access for [Brandon Cooper]

### ISSUE THREAD

**Issue by coope93** [department-of-veterans-affairs/va.gov-team#83763]
### Your Name

Brandon Cooper

### Your GitHub handle (terminal access is managed through GitHub)

coope93

### Team, Role, and Company of the target individual

Healthcare Enrollment, Senior Software Engineer, Ad Hoc

### Product Manager (PM) name and email

Heather Justice, heather.justice@adhocteam.us

### Product Owner (PO) name and email

Patrick Bateman, patrick.bateman@va.gov

### COR Name (leave "None" if you listed an AWS user name above)

Other - please specify in 'Additional Notes'

### Vendor Onboarding Representative (VOR) Name (leave "None" if you listed an AWS user name above)

Agile Six - Anthony M. Arashiro

### Please confirm if you are listed as a team member in Atlas or are listed in Platform Team Roster

- [X] [Team Member in Atlas](https://www.va.gov/atlas/product_directory/team_members)
- [ ] [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)

### Please confirm E-QIP Transmittal/Adjudication. This could be a screenshot of your Transmittal/Adjudication email, a link to another request where you've submitted this proof (i.e. AWS/SOCKS), etc.

- [X] I have attached proof that I have either transmitted my E-QIP survey or previously received a favorable decision (by including in "Additional Notes" section below)

### ArgoCD terminal access requested for the following environments

- [X] dev
- [X] staging
- [X] sandbox
- [X] production (if requesting, justify prod access in 'Additional Notes' section)

### Additional Notes

![Screenshot 2024-05-14 at 11 40 51 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/16926386/ba4361ad-3c12-476d-9dca-42a3bb9977ed)
- COR: Pamela Jones
- Requesting production access to investigate a ticket I am working on (https://github.com/department-of-veterans-affairs/va.gov-team/issues/81336). I am investigating the payload size and what we are sending, and we store a `PersonalInformationLog` that is in the local DB that has some data I would like to examine.

### Additional resources for Support Engineer

- [ ] For users on a VFS team but not listed as a team member in Atlas, they must start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] Instructions for handling requests [here](https://vfs.atlassian.net/wiki/spaces/PTST/pages/2183037084/Backend+Support+Responsibilities#Handle-vets-api-ArgoCD-terminal-access) - Support Only

**Comment by davidpetras** [comment]
@coope93  You’ve been granted access to the `dev`, `staging`, and `sandbox` environments.

Here's some additional documentation for [Vets API on EKS](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-TerminalAccess) that may help.

**Comment by davidpetras** [comment]
Access to `prod` is pending approval by OCTODE, and we'll update you as 
soon as it's available.

**Comment by davidpetras** [comment]
@coope93 If at all possible would you be able to give a little bit more detail on the ticket you are on working on and the reason `prod` access is needed. 

Thank you!

**Comment by JonathanKamensDVA** [comment]
Approved, but let's limit this to a month please since it's for a specific ticket and ask @coope93 to come back and let us know in a month if he still needs access. Thank you!

**Comment by davidpetras** [comment]
@coope93 your access to `production` has expired. Are you still in need of production access for the ticket you were working on?

Let me know here as prod access will be revoked by end of day if we do not hear from you.

**Comment by coope93** [comment]
I still need access for this to be able to trouble shoot and find logs when one of our apps has an error.

**Comment by davidpetras** [comment]
@JonathanKamensDVA would you be able to approve the extension requested here?

**Comment by JonathanKamensDVA** [comment]
@coope93 clarification please: have you found since May that you need prod access in general for debugging, aside from the debugging you were doing for the specific issue you were working on in May, or are you saying you want to extend your access to work more on this specific ticket? Thanks!

**Comment by coope93** [comment]
@JonathanKamensDVA Yes, I have found I need access pretty consistently to debug various errors in our applications. I work on the Health Apps team so we have a lot of different types of data being entered, and a few different API's that we interact with and have to handle different stuff coming our way. Having terminal access allows me to look into the errors and get Veteran details that allow our team to contact Veterans when we find something has gone wrong.

**Comment by JonathanKamensDVA** [comment]
Approved.

**Comment by davidpetras** [comment]
@coope93 your Production access has been extended for 6 months,
expiring 01/02/25.

Please set a reminder for yourself to return to this Issue and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 48 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by davidpetras** [comment]
@coope93  Your <Vets API Terminal> Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your 
access will be revoked and you will need to fill out a new request.

**Comment by coope93** [comment]
@davidpetras I would like an extension, thanks!

**Comment by davidpetras** [comment]
@coope93 your Production access has been extended for 6 months,
expiring 07/07/25.

Please set a reminder for yourself to return to this Issue and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 48 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by AnthonyWFlowers** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?
